docs/osf-trolling-list/build_process.md

```diff
@@ -0,0 +1,127 @@
+`[OSFI001]` Build process
+-------------------------
+
+# `[OSFI0010]` There is no healthy OSF without reproducible builds
+
+## What are the reproducible builds?
+
+[Reproducible Builds](https://reproducible-builds.org/) and lot of its aspects
+is well explained on linked website.
+
+## Why those are in particular important for OSF?
+
+This is because knowing hashes of firmware components we can calculate expected
+TPM PCRs values. Knowing good PCR values and being able to predict further
+values after firmware update, we gaining following advantages:
+
+* We can easily confirm if firmware is valid by reading TPM PCRs and compare to
```

Comment:
In my opinion: this is only if a TPM is open source hardware&firmware, only then it could be trusted for validation. If you agree, maybe also mention this somewhere.

Reply:
@mikebdp2 I think again it depends on the threat model. I think organizations associated with TCG and trusting OEM associated will not care much as it happens with Dell, HP, Microsoft, and many others. This is the note I added.

Reply:
I see there are some TPM products at 3mdeb shop - https://3mdeb.com/?s=TPM&post_type=product&dgwt_wcas=1 . @pietrushnic please tell, are their firmwares opensource? _(as of today)_ If yes, it's a good opportunity to mention them at this TPM note.

Reply:
@mikebdp2 no, unfortunately not, those are Infineon chips with closed source firmware.